examples/lsm303_inclinometer.py

```diff
@@ -0,0 +1,28 @@
+""" Display inclination data five times per second """
+
+import time
+from math import atan2, degrees
+import board
+import busio
+import adafruit_mpu6050
+
+i2c = busio.I2C(board.SCL, board.SDA)
+sensor = adafruit_mpu6050.MPU6050(i2c)
+
+
+def vector_2_degrees(x, y):
```

Comment:
Looks good, works well. 

My one request is that you add a comment explaining what `vector_2_degrees` is doing and why.

 This will be example code so it would be good if you could help someone not familiar with what is happening to understand how vectoring two degrees relates to inclination.

Thanks :)

Reply:
I've added some comments explaining the functions and providing a link to a learn guide page that explains the underlying principals well. I also had the filename incorrect so I've fixed that as well.